CHANGES.rst

```diff
@@ -1,10 +1,11 @@
 Changelog of fews-3di
 ===================================================
 
-3.1 (unreleased)
+
+3.0.1 (unreleased)
```

Comment:
Deze wijziging hoeft niet. Mag wel, _maar_ dan ook in `setup.py`, daar staat hetzelfde nummertje.